projectroles/views.py

```diff
@@ -2901,6 +2908,27 @@ def form_valid(self, form):
         )
         return redirect(reverse('projectroles:remote_sites'))
 
+    def create_timeline_event(
+        self, remote_site, user, form_action, timeline=None
+    ):
+        """Create timeline event for remote site creation/update"""
+        status = form_action if form_action == 'set' else form_action[0:-1]
+        if remote_site.mode == SITE_MODE_SOURCE:
+            event_name = 'source_site_{}'.format(status)
+        else:
+            event_name = 'target_site_{}'.format(status)
+        if timeline:
+            timeline.add_event(
+                project=None,
+                app_name=APP_NAME,
+                user=user,
+                event_name=event_name,
+                description='{} remote site "{}"'.format(
```

Comment:
The description could also contain whether the created site was a source or a target site.